x-pack/plugins/apm/public/store/urlParams.ts

```diff
@@ -19,6 +20,27 @@ import { IReduxState } from './rootReducer';
 
 // ACTION TYPES
 export const TIMEPICKER_UPDATE = 'TIMEPICKER_UPDATE';
+export const TIMEPICKER_DEFAULTS = {
+  rangeFrom: 'now-24h',
+  rangeTo: 'now',
+  rangePaused: 'true',
+  refreshInterval: '0'
+};
+
+function calculateTimePickerDefaults() {
+  const parsed = {
+    from: datemath.parse(TIMEPICKER_DEFAULTS.rangeFrom),
+    // roundUp: true is required for the quick select relative date values to work properly
+    to: datemath.parse(TIMEPICKER_DEFAULTS.rangeTo, { roundUp: true })
+  };
+
+  return {
+    start: parsed.from ? parsed.from.toISOString() : undefined,
+    end: parsed.to ? parsed.to.toISOString() : undefined
+  };
+}
+
+const INITIAL_STATE = calculateTimePickerDefaults();
```

Comment:
```suggestion
const INITIAL_STATE: IUrlParams = calculateTimePickerDefaults();
```

Reply:
Good idea, this required a little more type massaging so I took care of it. Thanks.